### ISSUE TITLE
how can I see the docstring of function 

### ISSUE THREAD

**Issue by zhiyiYo** [microsoft/pylance-release#2472]
## Environment data

-   Language Server version: 2022.3.2-pre.1
-   OS and version: Windows_NT x64 10.0.22000
-   Python version (& distribution if applicable, e.g. Anaconda): 3.8.6 64bit

## Expected behaviour
I can see both function signature and docstring
![image1](https://user-images.githubusercontent.com/65096819/158116979-a54c3062-5911-4fbe-9350-8bbb9854f3cb.png)

## Actual behaviour
I can only see the function signature.
![image](https://user-images.githubusercontent.com/65096819/158117124-c51b06c3-5cc4-4a7d-8595-d184acfd6355.png)

## Code Snippet 

```python
from PIL import Image
image = Image.open('XXX.png')
image.convert()
```

**Comment by judej** [comment]
Thanks for the report. Will investigate this.

**Comment by OmerFI** [comment]
The same as #2251

**Comment by bschnurr** [comment]
This issue has been fixed in version 2022.4.0, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202240-6-April-2022)

**Comment by zhiyiYo** [comment]
thx
